src/pages/LendingTeam/Leaderboards.vue

```diff
@@ -0,0 +1,229 @@
+<template>
```

Comment:
For tablet and mobile design has leaderboards displaying top 4 with button to expand to see more, this still needs to be implemented